content/guides/getting-started/component-framework-configuration.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: Framework Configuration
-containerClass: component-testing
 ---
 
 Recent years have seen an explosion in component based libraries (Vue, React)
```

Comment:
This paragraph can be refactored to just read:
```
Cypress component tests are written and behave the same regardless of framework choice (Vue, React, Nuxt, Next); however, some frameworks require additional configuration to work correctly. 
```

Reply:
Fixed in e42647b66fd24aaae85195822a567b9aedf78754